src/Icons/src/Svg/Icon.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace Symfony\UX\Icons\Svg;
+
+/**
+ *
+ * @author Simon André <smn.andre@gmail.com>
+ *
+ * @internal
+ */
+final class Icon implements \Stringable, \Serializable, \ArrayAccess
+{
+    /**
+     * Transforms a valid icon ID into an icon name.
+     *
+     * @throws \InvalidArgumentException if the ID is not valid
+     * @see isValidId()
+     */
+    public static function idToName(string $id): string
```

Comment:
we need to actually use these new methods now, right?

Reply:
Of course, i try to do micro step by micro step because the rebase dance is not my first talent :)